src/angular/datepicker/datepicker/datepicker.ts

```diff
@@ -154,6 +154,15 @@ export class SbbDatepicker<D> implements OnDestroy {
   }
   private _toggle = true;
 
+  /** Whether the datepicker toggle should be hidden. Defaults to false. */
+  @Input()
+  get noToggle(): boolean {
```

Comment:
Should we maybe have a setter only property? The state can easily be checked via `toggle`?